### ISSUE TITLE
macos app working good on debug mode but crash in release mode

### ISSUE THREAD

**Issue by HeroMonster** [flutter/flutter#72768]
building macos app working good on debug mode but crash in release mode and got this error.
Please help, thanks!

ERROR:flutter/runtime/dart_vm_data.cc(18)] VM snapshot invalid and could not be inferred from settings.
[ERROR:flutter/runtime/dart_vm.cc(250)] Could not setup VM data to bootstrap the VM from.
[ERROR:flutter/runtime/dart_vm_lifecycle.cc(84)] Could not create Dart VM instance.
[FATAL:flutter/shell/common/shell.cc(267)] Check failed: vm. Must be able to initialize the VM.

**Comment by TahaTesser** [comment]
Hi @HeroMonster 
This is a duplicate of https://github.com/flutter/flutter/issues/72585
Should be fixed once new Engine revisions lands on the  `master` channel 
Closing as fixed, f you disagree, please write in the comments and I will reopen it.
Thank you
